spec/system/admin/editing_an_instrument_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Editing an instrument" do
+  describe "daily booking instrument" do
+    let(:instrument) { create :setup_instrument, :daily_booking }
+    let(:facility) { instrument.facility }
+    let(:user) { create :user, :administrator }
+
+    before do
+      login_as user
+    end
+
+    it "can edit the instrument" do
+      visit edit_facility_instrument_path(facility, instrument)
+
+      expect(page).to have_content(instrument.name)
+
+      fill_in("Minimum (days)", with: "2")
+
+      click_button("Save")
+
+      expect(page).to have_content("Instrument was successfully updated")
+    end
+
+    context "switching start_time_disabled on" do
```

Comment:
What happens with reservations from before this change? I guess they'll work unless they want to update them. Should we check this with clients?

Reply:
I think that's fine, but may be worth it to add a testing note about it